charts/flink-operator/templates/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.mutatingWebhook.enabled }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Release.Name }}-serving-cert
+  name: {{ .Release.Name }}-mutating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: {{ .Release.Name }}-webhook-service
+      namespace: {{ .Release.Namespace }}
+      path: /mutate-flinkoperator-k8s-io-v1beta1-flinkcluster
+  failurePolicy: Fail
+  name: mflinkcluster.flinkoperator.k8s.io
+  {{- if .Values.namespaceScoped }}
+  namespaceSelector:
+    matchExpressions:
+      - key: flink-operator-namespace
```

Comment:
After poking around, I suggest changing this to `kubernetes.io/metadata.name`.. that label is automatically applied to namespaces and it matches the actual name of the namespace:

```
$ k get ns -l kubernetes.io/metadata.name=kafka
NAME    STATUS   AGE
kafka   Active   179d
```

Reply:
good find! changed it